### PR TITLE
stable/stolon Add Kubernetes 1.16+ support for Stolon cluster

### LINKED ISSUE #18412: [stable/stolon] does not start
**When start chart got error:**
Failed to install app stolon. Error: validation failed: [unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta2", unable to recognize "": no matches for kind "Deployment" in version "apps/v1beta2", unable to recognize "": no matches for kind "StatefulSet" in version "apps/v1beta2"]

**Version of Helm and Kubernetes**:
Helm: 2.14.3
Kubernetes Version: v1.16.2

**Which chart**:
Version 1.3.0

**What happened**:

**What you expected to happen**:
Start stolon-cluster

**How to reproduce it** (as minimally and precisely as possible):
It's default values except pass, i'm just started this chart.

**Anything else we need to know**:
I'm use rancher

### PR DESCRIPTION
**Pull request by sheremet**
# Add Kubernetes 1.16+ support for Stolon cluster

- add `kubernetes.version` variable for fixing deprecated `apiVersion` for Kubernetes 1.16+

- fixes #18412

### CODE REVIEW

**Review by joschi36 — APPROVED** [review]

File: stable/stolon/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: stolon
-version: 1.4.0
+version: 1.5.0
```
Comment (Negashev) [inline stable/stolon/Chart.yaml:3]: Please, pull master and update version to 1.6.0

**Pull request closed without merging**